csharp/src/Microsoft.ML.OnnxRuntime/NativeMethods.cs

```diff
@@ -174,6 +174,7 @@ public struct OrtApi
         public IntPtr GetBoundOutputValues;
         public IntPtr ClearBoundInputs;
         public IntPtr ClearBoundOutputs;
+        public IntPtr SetLanguageProjection;
```

Comment:
>SetLanguageProjection [](start = 22, length = 21)

There are two entry points were added after ClearBoundOutputs, the must be added here before your new API